aspnetcore/tutorials/first-mongo-app/first-mongo-app.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Build Web APIs with ASP.NET Core and MongoDB
+author: pratik-khandelwal
+description: Get an introduction to MongoDB, an open-source, non-relational database and how to build APIs using MongoDBand ASP.NET Core.
+uid: tutorial/first-mongo-app
+---
+# Build Web APIs with ASP.NET Core and MongoDB
+
+By [Pratik Khandelwal](https://twitter.com/K2Prk)
+
+### What is MongoDB?
+
+MongoDB is an open-source, non-relational database developed by MongoDB, Inc. MongoDB stores data as documents in a binary representation called BSON (Binary JSON). In modern web applications, the use of NoSQL databases is becoming popular due to it's non-relational bahaviour.
+
+In this tutorial, we will create a Web API using ASP.NET Core to perform CRUD operations on a MongoDB Collection.
+
+### Setting up MongoDB
+
+To get started, you first need to install MongoDB on your system. You can download MongoDB from here. If you are on Windows, MongoDB is installed in the following path
+
+C:\Program Files\MongoDB
+
+Also, don't forget to add **C:\Program Files\MongoDB\Server\3.4\bin** to Environment Path to access MongoDB from anywhere.
+
+### MongoDB Implementation
+
+To use MongoDB, we have to create a DB, make collections and store documents in it. To do that, we need to create a a data folder where your data will be stored. You can choose any location on your system as the data folder. (Keep note that MongoDB CLI does not create new Directories).
+
+Open the Powershell or command promt and run the following command
```

Comment:
~~Open the Powershell or command promt~~
Open a command shell

This sentence should end with a colon.